packages/react_rearch/lib/src/widgets/capsule_container_provider.dart

```diff
@@ -0,0 +1,3 @@
+part of '../widgets.dart';
+
+final topLevelCapsuleContainer = CapsuleContainer();
```

Comment:
Could this be done via the React context API? https://legacy.reactjs.org/docs/context.html

Then we can have one top-level component that provides the container, similar to CapsuleContainerProvider, and another than disposes the container on component disposal.

Reply:
Didn't knew this but seems the appropiated tool to have a CapsuleContainerProvider

Reply:
https://github.com/GregoryConrad/rearch-dart/discussions/150#discussioncomment-9166017

Reply:
Done